list2/ex32/index.js

```diff
@@ -0,0 +1,38 @@
+import { generateMatrix, printMatrix } from "../helper.js";
+
+export function matrixRowNormalization() {
+  const rowCount = 12;
+  const columnCount = 13;
+
+  const matrix = generateMatrix({
+    rowCount,
+    columnCount,
+  });
+
+  printMatrix(matrix);
+  console.log("---------------");
+
+  function newMatrix({ matrix, rowCount, columnCount }) {
+    const modifiedMatrix = JSON.parse(JSON.stringify(matrix));
+
+    for (let rowIndex = 0; rowIndex < rowCount; rowIndex++) {
+      const biggestValue = Math.max(...modifiedMatrix[rowIndex]);
```

Comment:
The code calculates the largest value using `Math.max(...)`, but it does not consider the absolute value of the elements. The requirement specifies that the division should be done by the largest absolute value in each row. Please, fix this.